tests/sql/query.yaml

```diff
@@ -17,6 +17,8 @@ chapters:
   - synopsis: Get SQL query
     path: /_plugins/_sql
     method: POST
+    parameters:
+      sanitize: false
```

Comment:
as you mentioned at previous PR @dblock
i added parameters with sanitize for one test